provisioning/install-precice.sh

```diff
@@ -24,6 +24,7 @@ cd /home/${USER}/Desktop
         sudo -u ${USER} git clone --branch master https://github.com/precice/tutorials.git
     fi
 cd -
+sudo apt-get -y install gnuplot # needed for watchpoint scripts of tutorials
```

Comment:
```suggestion
apt-get -y install gnuplot # needed for watchpoint scripts of tutorials
```
In Vagrant, while provisioning, we don't need `sudo` to install software. Above I use `sudo -u` to execute as a specific user (which, in some cases, may not be needed either actually).